bundles/org.openhab.binding.enturno/README.md

```diff
@@ -64,34 +64,34 @@ demo.items
 
 ```java
 // Stop place
-String      StopId                          "StopId"                        {channel="enturno:linestop:1:stopPlace#id"}               
-String      StopPlaceName                   "Stop Place [%s]"               {channel="enturno:linestop:1:stopPlace#name"}
-String      LineCode                        "Line [%s]"                     {channel="enturno:linestop:1:Direction01#lineCode"} 
-String      TransportMode                   "TransportMode [%s]"            {channel="enturno:linestop:1:stopPlace#transportMode"} 
+String      StopId                          "StopId"                        {channel="enturno:linestop:7e693fff:stopPlace#id"}
+String      StopPlaceName                   "Stop Place [%s]"               {channel="enturno:linestop:7e693fff:stopPlace#name"}
+String      LineCode                        "Line [%s]"                     {channel="enturno:linestop:7e693fff:Direction01#lineCode"}
+String      TransportMode                   "TransportMode [%s]"            {channel="enturno:linestop:7e693fff:stopPlace#transportMode"}
 
 // Direction01
-String      Direction01_FrontDisplay        "Direction01 front display [%s]"    {channel="enturno:linestop:1:Direction01#frontDisplayText"} 
-DateTime    RealTime_Departure01_Time1      "Departure01 time"                  {channel="enturno:linestop:1:Direction01#departure01"}     
```

Comment:
Not sure why the item name is changed. It's just a user given name.

Reply:
Renaming reflects better what the thing is and it's connection to things. But what is more important these names will be used in widget I'm creating so it will work out of the box without unnecessary adjustments.